miden/tests/integration/stdlib/mem.rs

```diff
@@ -0,0 +1,49 @@
+pub use math::fields::f64::BaseElement;
+pub use math::FieldElement;
+use miden::MemAdviceProvider;
+use processor::{AdviceInputs, Process, StackInputs};
+use stdlib::StdLibrary;
+
+#[test]
+fn memcopy() {
+    let source = "
+    use.std::mem
+
+    begin
+        push.0.0.0.1.1000 mem_storew dropw
+        push.0.0.1.0.1001 mem_storew dropw
+        push.0.0.1.1.1002 mem_storew dropw
+        push.0.1.0.0.1003 mem_storew dropw
+        push.0.1.0.1.1004 mem_storew dropw
+
+        push.5.2000.1000 exec.mem::memcopy
+    end
+    ";
+
+    let assembler = assembly::Assembler::default()
+        .with_library(&StdLibrary::default())
+        .expect("failed to load stdlib");
+
+    let program = assembler.compile(source).expect("Failed to compile test source.");
+
+    let mut process = Process::new(
+        program.kernel().clone(),
+        StackInputs::default(),
+        MemAdviceProvider::from(AdviceInputs::default()),
+    );
+    process.execute(&program).unwrap();
```

Comment:
The `Test` helper didn't have a method to inspect multiple memory addresses. It was easier to instantiate the dependencies in the test